airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/limit_paginator.py

```diff
@@ -78,15 +78,17 @@ def __init__(
         config: Config,
         url_base: str,
         decoder: Decoder = None,
+        **options: Optional[Mapping[str, Any]],
```

Comment:
trying to onfirm my understanding, because we now have the `options` param, when we invoke `inspect.getfullargspec(func)` in `create_partial.py`, we will now receive the `$options` whereas before we dropped them since they were not part of the input params and would thus get filtered out. 

I might be misunderstanding, but wouldn't that mean every concrete component would need to have `**options` appended for them to receive options, not just the one limit_paginator?

Reply:
I only updated the components that create `InterpolatedString`.
The options are passed down to the subcomponents even if the parent doesn't have an `options` parameter from https://github.com/airbytehq/airbyte/blob/982f5009d48b6ee3450b07697d8511b4e46147e8/airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/factory.py#L144

that being said, I agree that it would make sense to always have it as parameter for consistency.

Will update.

Reply:
ah i see thanks for clarifying. updating for consistency sounds good